src/tools/annotation/bidirectionalTool/moveHandle/moveLongLine/getDistanceToIntersection.test.js

```diff
@@ -0,0 +1,44 @@
+import getDistanceToIntersection from './getDistanceToIntersection';
+
+function createPoint(x, y) {
+  return {
+    x,
+    y,
+  };
+}
+
+describe('getDistanceToIntersection.js', () => {
+  it('distance is positive', () => {
+    const columnPixelSpacing = 1;
+    const rowPixelSpacing = 0.5;
+    const intersection = createPoint(1, 1);
+    const point = createPoint(5, 5);
+
+    const baseData = {
+      columnPixelSpacing,
+      rowPixelSpacing,
+      intersection,
+    };
+
+    const distance = getDistanceToIntersection(baseData, point);
+
+    expect(distance).toEqual(8.94427190999916);
```

Comment:
Is that float precision? I think you want to use `.toBeCloseTo` https://jest-bot.github.io/jest/docs/expect.html#tobeclosetonumber-numdigits

Reply:
This test was also deleted as impact of refactoring.
But I'll observe it in the other tests and change them to use this assertion.
Thank you!